### ISSUE TITLE
Printing randomly stops

### ISSUE THREAD

**Issue by Silvood** [UP3D-gcode/UP3D#12]
Hello,

thanks for the wonderful software!
I'm having some issues with the printer itself which could be caused by he code generated with this tool.
The printing is starting correctly, but after some minutes, it just stop for no reason at all. Tried 3 times, stopped 3 times during raft, just after raft and during the 3rd layer of the piece.
Here is the up3dshell screen after the stop occurs.
![2017-03-05](https://cloud.githubusercontent.com/assets/5123279/23587096/c718a80c-01a4-11e7-90f6-b29ba261f5b0.png)

If someone know a solution to solve this, he'll be thanks :)

Nota: I'm running it on windows10 x64, with simplify3D, and a up mini.

Edit: tried with build plateform temp 60, same result :/

Edit bis: related to https://github.com/MaikStohn/UP3D/issues/26

**Comment by kcarbuncle** [comment]
Same problem, the printer just stopped and also extruded a blob of material before stopping.

Using the latest version from here, and uploaded a screenshot of the settings + gcode/umc file for diagnosis. Printer is an UP Plus 2 (TypeID: 277b, rom version: 6.09000).

Also, dropping down the a_factor to 30 (or below) and v_factor to 70 (and below) and the print was successful.

[photo](https://cloud.githubusercontent.com/assets/28801721/26237672/b4cbcad2-3ca8-11e7-9402-d21bee42b345.JPG)

[Error files.zip](https://github.com/UP3D-gcode/UP3D/files/1013460/Error.zip)

**Comment by halfluck** [comment]
@kcarbuncle are you able to offer any suggestions or a code snippet on how you implemented he a_factor and v_factor changes?

Many Thanks Rob.

**Comment by kcarbuncle** [comment]
@halfluck : Not really, just reporting that the converter and maybe the loader does not like fast movements, and at times it even feels like random (even with the fine/slow settings).

Something is clearly crashing the UP when it reads a line from the converted/uploaded gcode...

**Comment by non-bin** [comment]
Was there ever a fix for this? I have the same problem with my Plus
